frontend/src/api/api.tsx

```diff
@@ -366,6 +366,14 @@ const addTaskRelation = async (relation: TaskRelationRequest) => {
   return response.status === 200;
 };
 
+const removeTaskRelation = async (relation: TaskRelationRequest) => {
+  const response = await axios.delete(
+    `/roadmaps/${relation.roadmapId}/tasks/relations`,
+    { data: { from: relation.from, to: relation.to, type: relation.type } },
```

Comment:
Is there a reason the relation is nested like this?
```suggestion
    { from: relation.from, to: relation.to, type: relation.type },
```
Or if the TaskRelationRequest type is changed to TaskRelation
```suggestion
    relation,
```

Reply:
Axios.delete requires you to nest data like that if you want to give it body params. I cleaned it up a bit though, now it has ` { data: relation },` instead of `{ data: { from: relation.from, to: relation.to, type: relation.type } },`

_axios.delete does support a request body. It accepts two parameters: url and optional config. You can use config.data to set the request body and headers as follows: ..._
https://stackoverflow.com/questions/51069552/axios-delete-request-with-body-and-headers